smart-contracts/test/Lock/erc721/getApproved.js

```diff
@@ -1,18 +1,19 @@
 const deployLocks = require('../../helpers/deployLocks')
 const Unlock = artifacts.require('../../Unlock.sol')
+const Zos = require('zos')
+const TestHelper = Zos.TestHelper
 
 let unlock, locks
 
 contract('Lock ERC721', (accounts) => {
-  before(() => {
-    return Unlock.deployed()
-      .then(_unlock => {
-        unlock = _unlock
-        return deployLocks(unlock)
-      })
-      .then(_locks => {
-        locks = _locks
-      })
+  const proxyAdmin = accounts[1]
+  const unlockOwner = accounts[2]
+
+  before(async function () {
+    const project = await TestHelper({ from: proxyAdmin })
+    const proxy = await project.createProxy(Unlock, { initMethod: 'initialize', initArgs: [unlockOwner], initFrom: unlockOwner })
```

Comment:
will do.

Reply:
Fixed. Switched to `this.project` and `this.proxy`.

Reply:
@julien51 Did you want an explanation of what `project` and `proxy` are, or why I used `const project` instead of `this.project`? I've changed to `this.project`, etc  for consistency.

Reply:
Also commented in UnlockProxy.js about this, as it's the place we use `TestHelper` the most,